xray/commands/audit/audit.go

```diff
@@ -87,14 +87,18 @@ func (auditCmd *AuditCommand) Run() (err error) {
 	if err != nil {
 		return
 	}
+	// If no workingDirs were provided by the user, we apply a recursive scan on the root repository
```

Comment:
wrong place, should be before line 86 where it gets the root working dir if it is empty.